lib/spreadsheet_architect/class_methods/xlsx.rb

```diff
@@ -30,7 +30,7 @@ def to_axlsx_package(opts={}, package=nil)
       package.workbook.add_worksheet(name: options[:sheet_name]) do |sheet|
         max_row_length = options[:data].empty? ? 0 : options[:data].max_by{|x| x.length}.length
 
-        if options[:headers]
+        if options[:headers].flatten.any?
```

Comment:
I could have updated https://github.com/westonganger/spreadsheet_architect/blob/master/lib/spreadsheet_architect/utils.rb#L10 with some kind of tweak to fix this though I think it's nice and consistent that options[:headers] is always an array.